xds/internal/server/listener_wrapper.go

```diff
@@ -301,34 +302,37 @@ func (l *listenerWrapper) Accept() (net.Conn, error) {
 			conn.Close()
 			continue
 		}
-		var rc xdsclient.RouteConfigUpdate
-		if fc.InlineRouteConfig != nil {
-			rc = *fc.InlineRouteConfig
-		} else {
-			rcPtr := atomic.LoadPointer(&l.rdsUpdates)
-			rcuPtr := (*map[string]xdsclient.RouteConfigUpdate)(rcPtr)
-			// This shouldn't happen, but this error protects against a panic.
-			if rcuPtr == nil {
-				return nil, errors.New("route configuration pointer is nil")
+		var vhswi []xdsclient.VirtualHostWithInterceptors
+		if env.RBACSupport {
```

Comment:
Explained thought process in chat, I think it's best to ignore any route configuration and only explicitly validate a configured top level RBAC HTTP Filter, as per the example discussed with Srini of a user configuring an RBAC Filter and have it working with Java, but not in Go, so they should definitely know that since it's a security question.

Reply:
Nit:

Instead of :
```golang
if env.RBACSupport {
  // Huge block of code
}
return <something>
```

```golang
if !env.RBACSupport {
  // return early
}
// Huge block of code
```

Reply:
Switched.